main.html

```diff
@@ -5,50 +5,53 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="main.css" rel="stylesheet">
-    <title>Presentation</title>
-    <style>
-        @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
-        </style>
+    <title>Document</title>
 </head>
 <body>
-    <div class="top-navigation">
-        <div class="item-navigation">
-            <div class="frame-4">
-                <p>Portfolio</p>
-            </div>
-            <div class="frame-3">
-                <p>About</p>
-            </div>
-            <div class="frame-5">
-                <p>Contact</p>
-            </div>
+    <div class="container">
+    <header class="my-header">
+        <div class="time">
+            <img src="mobile_menu_images/Bars/Status-Bars/Time-Style.jpg" alt="Header image">
         </div>
-        <div class="logo-placeholder">
-            <p>My Logo</p>
+        <div class="status-bar">
+            <img src="mobile_menu_images/Bars/Status-Bars/Group3.jpg" alt="Bar Status icons">
         </div>
-    </div>
+    </header>
```

Comment:
![image](https://user-images.githubusercontent.com/80612925/170714225-7368abe8-6310-40a5-a967-b21924f9f9db.png)

- [ ] I would suggest you remove this part from your code as it's just a simple illustration of how your portfolio could look on mobile. You can start by the `menu` bar as required.